config/am-scripts/scripts-content/ch-join-company-create-relationship.js

```diff
@@ -179,6 +177,7 @@ if (!JSON.parse(companyData).authCodeIsActive) {
             'companyNumber': JSON.parse(companyData).name
```

Comment:
Is this required and if so, is this supposed to be outputting companyData.name instead of company number?